### ISSUE TITLE
Añadir a f-droid

### ISSUE THREAD

**Issue by wuniversales** [juanro49/FeedTV#20]
Segun pone en el readme, FeedTV está disponible para f-droid, pero no es asi...
¿Podria ser añadido? o esta en proceso..

**Comment by juanro49** [comment]
> Segun pone en el readme, FeedTV está disponible para f-droid, pero no es asi...
> ¿Podria ser añadido? o esta en proceso..

Está disponible a través del repo de IzzyOnDroid, por lo que para obtenerlo en F-droid tienes que añadir anteriormente este repositorio.

https://apt.izzysoft.de/fdroid/repo?fingerprint=3BF0D6ABFEAE2F401707B6D966BE743BF0EEE49C2561B9BA39073711F628937A

**Comment by wuniversales** [comment]
Ahh, vale, ya está instalado...
¿Pero va a ser añadido en ell repositorio oficial?
Seria lo suyo XD

**Comment by juanro49** [comment]
> Ahh, vale, ya está instalado...
> ¿Pero va a ser añadido en ell repositorio oficial?
> Seria lo suyo XD

Pensé añadirlo, pero en el repositorio oficial compilan ellos mismos las versiones y los apk que generan tienen una firma diferente, y la idea es distribuir el mismo apk en todas las plataformas para evitar problemas a la hora de actualizar

**Comment by wuniversales** [comment]
Claro, lo compilan directamente del código de fuente, de la ultima versión.
Es un poco cuestión de etica, los amantes del open source prefieren descargar directamente de la repo de f-droid, tal vez poniendo una aclaración con lo de las firmas en el readme se evite ese problema.
